public/css/style.css

```diff
@@ -430,7 +430,7 @@ li code {
     justify-content: space-between;
 }
 .integration {
-    width: 33.3333%;
+    width: 25%;
```

Comment:
Nice!
We can also talk about dropping Dust from the home page, but keeping it listed (last) on the Guide and Integrations pages. But that can be tweaked in another PR.